docs/release_notes/v1.13.0.md

```diff
@@ -0,0 +1,519 @@
+The component spec is reconciled and takes effect when running in both Kubernetes and Self-Hosted modes.
+
+### Go and JavaScript/TypeScript SDK support for workflow
+
+You can now use Go and JavaScript/TypeScript to write fault-tolerant, workflow-based apps with durable execution.
+See the [Go SDK for more details](https://github.com/dapr/go-sdk/tree/main/examples/workflow) and try the [Quickstart](https://github.com/dapr/quickstarts/tree/release-1.13/workflows/go/sdk) or the [examples](https://github.com/dapr/go-sdk/blob/main/examples/workflow/README.md).
+See the [Javascript SDK for more details](https://github.com/dapr/js-sdk/tree/main/examples/workflow/authoring) and try the [Quickstart](https://github.com/dapr/quickstarts/tree/release-1.13/workflows/javascript/sdk) or the [examples](https://github.com/dapr/js-sdk/tree/main/examples/workflow)
+
+### Low metrics cardinality for HTTP server
+
+Dapr now includes an optional setting to enable low cardinality for metrics emitted by the HTTP server, which will be enabled by default in Dapr 1.14.
+
+Currently, the Dapr HTTP server emits metrics for each request path, which has been observed to cause significant memory usage and have other negative performance impact.
+When low-cardinality metrics are enabled, the HTTP Dapr server behaves more like the gRPC server and groups requests for each Dapr API together, possibly reducing memory consumption significantly.
+See additional info [here](https://v1-13.docs.dapr.io/operations/observability/metrics/metrics-overview/#high-cardinality-metrics).
+
+### Graceful shutdown
+
+Dapr now accepts a `dapr.io/block-shutdown-duration` annotation or `--dapr-block-shutdown-duration` CLI flag which delays the full shutdown procedure for this duration or until the app reports as unhealthy- whichever is sooner.
+During this period, all Subscriptions and input bindings will be closed.
+This is useful for applications that need to use the Dapr APIs as part of their own shutdown procedure.
+See additional info [here](https://v1-13.docs.dapr.io/reference/arguments-annotations-overview/).
+
+### Standardized error codes
+
+The Dapr PubSub and State APIs now have the appropriate and standardized error codes returned to applications including enriched error details based on gRPC's richer error model.
+The remaining APIs are a work in progress, and community contributions towards this effort are greatly appreciated and very impactful.
+See additional info on the error codes [here](https://docs.dapr.io/reference/api/error_codes/).
+Check the docs for each SDK for per-SDK error code parsing and handling.
+See an [example of the Go SDK error parsing and handling here](https://v1-13.docs.dapr.io/developing-applications/sdks/go/go-client/#error-handling).
+
+### Actor reminders performance improvement
```

Comment:
consider raising up the list.  perhaps after workflow?

Reply:
:+1: